api/app/utils.py

```diff
@@ -1,7 +1,11 @@
+import json
 import pathlib
 
 import shortuuid
 
+UNKNOWN = "unknown"
```

Comment:
There's a hardcoded `"unknown"` string at the bottom of this file too. Can we update all usages to use this constant?

Reply:
OK that's dine